tests/self_hosted.txt

```diff
@@ -9,9 +9,18 @@ tests/escape.oomph
 tests/float2int.oomph
 tests/hello.oomph
 tests/if.oomph
+tests/indent2_error.oomph
+tests/indent_error.oomph
+tests/indent_increase_error.oomph
+tests/invalid_token_error.oomph
+tests/leading_indent_error.oomph
+tests/leading_indent_with_comment_error.oomph
```

Comment:
This list is becoming huge. Might be better to switch to exclude list rather than include list

Reply:
#96